cmd/minikube/cmd/status.go

```diff
@@ -196,10 +197,12 @@ var statusCmd = &cobra.Command{
 	Exit status contains the status of minikube's VM, cluster and Kubernetes encoded on it's bits in this order from right to left.
 	Eg: 7 meaning: 1 (for minikube NOK) + 2 (for cluster NOK) + 4 (for Kubernetes NOK)`,
 	Run: func(cmd *cobra.Command, args []string) {
-		if output != "text" && statusFormat != defaultStatusFormat {
+		if strings.ToLower(output) != "text" && statusFormat != defaultStatusFormat {
```

Comment:
Instead of calling strings.ToLower(output) multiple times, let's say

```
output = strings.ToLower(output)
```
above this line

Reply:
nice catch, thanks @priyawadhwa, i've changed that